docs/01-Configuration.md

```diff
@@ -302,7 +302,7 @@ This example will not work as is, you'll need to supply your own values.
 keystore = "/some/path/keystore"
 
 [cosmos]
-gas_adjustment = 1.0
+gas_adjustment = 1.1
```

Comment:
If we're recommending this for the configuration, should we just set the default to 1.1?